beta1/integrated-build.json

```diff
@@ -93,8 +93,8 @@
         "strategy": {
           "type": "STI",
           "stiStrategy": {
-            "image": "openshift/ruby-20-centos",
-            "scripts": "https://raw.githubusercontent.com/openshift/ruby-20-centos/master/.sti/bin"
+            "image": "openshift/ruby-20-centos7",
+            "scripts": "https://raw.githubusercontent.com/openshift/sti-ruby/master/2.0/.sti/bin"
```

Comment:
If the build process is the same, why is there a change in the scripts folder? Does the old script location still exist in the new docker image?

Remember, we're talking about this in the context of `openshift ex generate`, which, for beta1, is going to reference `openshift/ruby-20-centos` and `https://raw.githubusercontent.com/openshift/ruby-20-centos/master/.sti/bin`

@mfojtik  @soltysh  @jhadvig